extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfLogHandlerRecorder.java

```diff
@@ -26,6 +27,31 @@
         handler.setTimestampPattern(config.timestampPattern);
         handler.setHost(config.host);
         handler.setPort(config.port);
+        handler.setLevel(config.level);
+
+        // handle additional fields
+        if (!config.additionalField.isEmpty()) {
+            StringBuilder additionalFieldsValue = new StringBuilder();
+            StringBuilder additionalFieldsType = new StringBuilder();
+            for (Map.Entry<String, AdditionalFieldConfig> additionalField : config.additionalField.entrySet()) {
+                if (additionalFieldsValue.length() > 0) {
+                    additionalFieldsValue.append(',');
+                }
+                additionalFieldsValue.append(additionalField.getKey()).append('=').append(additionalField.getValue().value);
+
+                additionalField.getValue().type.ifPresent(
+                        type -> {
```

Comment:
We try to avoid lambdas in runtime module

Reply:
done